app/models/guide_republisher.rb

```diff
@@ -12,7 +12,7 @@ def republish
 
       publishing_api.put_content(guide.content_id, guide_for_publication.content_payload)
```

Comment:
But this script is meant to re-publish guides but always as a minor edition, as we use this script if e.g. we need to change the way a guide is presented, but we don't want to trigger email alerts or show a change in the history of the guide.